plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraTypes.java

```diff
@@ -48,7 +42,7 @@ private CassandraTypes() {}
     public static final CassandraType DECIMAL = new CassandraType(Kind.DECIMAL, DoubleType.DOUBLE);
     public static final CassandraType DOUBLE = new CassandraType(Kind.DOUBLE, DoubleType.DOUBLE);
     public static final CassandraType FLOAT = new CassandraType(Kind.FLOAT, RealType.REAL);
-    public static final CassandraType INET = new CassandraType(Kind.INET, createVarcharType(IP_ADDRESS_STRING_MAX_LENGTH));
+    public static CassandraType INET; // Will be initialized in CassandraMetadata
```

Comment:
This comment is rather confusing.
Why would we need to do this? Take into account also the fact that multiple requests are handled in the same time or over the lifetime of Trino process, so this field would be initialized over and over multiple times.

Reply:
@findinpath You're right, I didn't realize it would be initialized over and over again.
I just want to initialize it once, but couldn't find an approach to do it gracefully. I need to call `typeManager().getType()` to get the type reference, but that doesn't seem to be possible in a static context like this..

Reply:
@findinpath I just removed the static `INET` type altogether. Please check the update if possible, thanks!